decoder/pokemon.go

```diff
@@ -724,8 +724,10 @@ func (pokemon *Pokemon) addEncounterPokemon(ctx context.Context, db db.DbDetails
 	var level int64
 	if cpMultiplier < 0.734 {
 		level = int64(math.Round((58.35178527*cpMultiplier-2.838007664)*cpMultiplier + 0.8539209906))
-	} else {
+	} else if cpMultiplier < .795 {
 		level = int64(math.Round(171.0112688*cpMultiplier - 95.20425243))
+	} else {
+		level = int64(math.Round(199.99995231630976*cpMultiplier - 118.05996066890287))
```

Comment:
I am just curious who is doing those calculations? cpMultiplier is listed in Gamemaster, but where do the other values come from? :)

Reply:
Asked chatgpt to write an interpolation. I can't be bothered to read what interpolation it is but I can send you the source code. (I also wrote tests to make sure it works.)